<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDlg</class>
 <widget class="QDialog" name="GameDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;
</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
   </property>
   <property name="text">
    <string>当前得分：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>200</y>
     <width>111</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
   </property>
   <property name="text">
    <string>得分进度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_score">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>130</y>
     <width>111</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_66">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_67">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
